notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -2932,6 +2932,44 @@ biological computer [Dr. Yuhara explained that Mechagodzilla was installed with
 nuclear weapons [Sara stated that it was the use of the H-bomb the first attracted Godzilla to Japan.]
 
 
+movie: Its All About Love (2003)
+================================
+
+:: Title
+It's All About Love
+
+:: Description
+It's All About Love is a 2003 English-language Danish romance-drama film
+written and directed by Thomas Vinterberg.
+
+Synopsis: A husband and wife resolve to save their relationship as the world
+is entering into an ice age.
+
+:: Date
+2003-01-10
+
+:: References
+https://en.wikipedia.org/wiki/It%27s_All_About_Love
+
+:: Ratings
+3 <paul>
+
+:: Choice Themes
+romantic love [The story follows John as he rekindled his relationship with his estranged, figure skating superstar wife Elena. As the title suggests, the story is above all a story about love and how it is the most important thing.]
+
+:: Major Themes
+husband and wife [The story follows John as he rekindled his relationship with his estranged, figure skating superstar wife Elena.]
+human cloning [Central to the plot was the figure skater Elena's entourage making a trio of Elena clones with which they plotted to replace her.]
+sports [The main character Elena was a superstar figure skater. The viewer is show her performing, and the sheltered, stage managed live that she lived.]
```

Comment:
show -> shown

Reply:
There's something stilted about that list, by the way. "Her performing" and "her sheltered life" are both conceptually and grammatically very different things.

Reply:
Indeed. That is a bit odd. I improved the motivation.